libpod/runtime.go

```diff
@@ -30,6 +30,7 @@ const (
 	// reboot
 	InMemoryStateStore RuntimeStateStore = iota
 	// SQLiteStateStore is a state backed by a SQLite database
+	// It is presently disabled
```

Comment:
Any reason to just delete this now?  I'm not finding other references in the code other than what you've removed already here.

Reply:
oops.  just == not just.  Basically, can we just delete this now rather than commenting/no-oping it?